packages/manager/apps/dedicated/client/app/billing/autoRenew/actions/autorenew-blocked/autorenew-blocked.module.js

```diff
@@ -0,0 +1,11 @@
+import component from './autorenew-blocked.component';
+import routing from './autorenew-blocked.routing';
+
+const moduleName = 'ovhManagerBillingAutorenewBlocked';
+
+angular
+  .module(moduleName, ['ui.router'])
```

Comment:
`atInternet` seems used but not declared as a dependency

Reply:
This is fixed